### ISSUE TITLE
Hostfolder mount fails with sshfs if 2022 port not open on the host firewalld

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#2506]
```
$ sudo iptable-save
[...]
-A IN_FedoraWorkstation_allow -p tcp -m tcp --dport 22 -m conntrack --ctstate NEW -j ACCEPT

$ ./minishift hostfolder list
Name   Type    Source   Mountpoint       Mounted
test   sshfs   /home/prkumar/Documents     /mnt/sda1/test   N
```
We should improve the below error message 
```
$ ./minishift hostfolder mount test
error occurred while mounting host folder: error occured while mounting host folder: ssh command error:
command : sudo sshfs docker@192.168.42.1:/home/prkumar/Documents /mnt/sda1/test -o IdentityFile=/home/docker/.ssh/id_rsa -o 'StrictHostKeyChecking=no' -o reconnect -o allow_other -o idmap=none -p 2022
err     : exit status 1
output  : read: Connection reset by peer

```
```
# iptables -A IN_FedoraWorkstation_allow -p tcp -m tcp --dport 2022 -m conntrack --ctstate NEW -j ACCEPT

# iptables-save
$ ./minishift hostfolder mount test
$ ./minishift hostfolder list
Name    Type    Source                    Mountpoint        Mounted
test    sshfs   /home/prkumar/Documents   /mnt/sda1/test    Y
```

**Comment by anjannath** [comment]
You can also change/set the port for the SFTP server, there's a config option for this: `hostfolders-sftp-port` I think the docs already mention this.

**Comment by praveenkumar** [comment]
@anjannath but whatever port you chose should be allowed by the firewall, does docs mention this, if yes link please?

**Comment by anjannath** [comment]
@praveenkumar Not with instructions for how to enable it. Just a _Tip_ mentioning that firewall should allow comm. to this port.. I can't link to it as it is not a section.. it under this section: 
https://docs.openshift.org/latest/minishift/using/host-folders.html#mounting-host-folders

**Comment by LalatenduMohanty** [comment]
Resolved #2517
